ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -927,78 +981,78 @@ define([
       return undefined
     },
 
-    // Format the internal representation of UTM coordinates into the form expected by the model.
-    formatUtm: function(utmCoords) {
+    // Format the internal representation of UTM/UPS coordinates into the form expected by the model.
+    formatUtmUps: function(utmUps) {
       return {
-        easting: utmCoords.easting,
-        northing: utmCoords.northing,
-        zoneNumber: utmCoords.zoneNumber,
+        easting: utmUps.easting,
+        northing: utmUps.northing,
+        zoneNumber: utmUps.zoneNumber,
         hemisphere:
-          utmCoords.hemisphere === 'NORTHERN'
+          utmUps.hemisphere === 'NORTHERN'
             ? 'Northern'
-            : utmCoords.hemisphere === 'SOUTHERN'
+            : utmUps.hemisphere === 'SOUTHERN'
               ? 'Southern'
               : undefined,
       }
     },
 
-    // Return true if all of the utm upper-left model fields are defined. Otherwise, false.
-    isUtmUpperLeftDefined: function() {
+    // Return true if all of the UTM/UPS upper-left model fields are defined. Otherwise, false.
+    isUtmUpsUpperLeftDefined: function() {
```

Comment:
✏️ as this file is already massive, consider moving some of these function into a separate utility file and importing them instead.

Reply:
we decided that this is out of scope for this ticket